### PR TITLE
Add - spandsp a library needed to setup fax capabilities

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

File: components/multimedia/spandsp/Makefile [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+
+BUILD_BITS=	64
+
```
Comment (AndWac) [inline components/multimedia/spandsp/Makefile]: BUILD_BITS= 64 is our default now an can be omitted.

File: components/multimedia/spandsp/Makefile [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+COMPONENT_ARCHIVE_HASH= sha256:fdee8fbba0b471500928cd88207a96e1eb89bbca6b518249cf58c11b767de2a1
+COMPONENT_PROJECT_URL=	https://github.com/freeswitch/spandsp
+COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/archive/refs/heads/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=		library/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= System/Multimedia Libraries
+COMPONENT_LICENSE=	LPGPLv2.1
+COMPONENT_LICENSE_FILE= COPYING
+
+TEST_TARGET=	$(NO_TESTS)
+
```
Comment (AndWac) [inline components/multimedia/spandsp/Makefile:35]: There seem to be tests in the sources. Have you tried to run them?

File: components/multimedia/spandsp/Makefile [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_PRE_CONFIGURE_ACTION  = ( \
```
Comment (AndWac) [inline components/multimedia/spandsp/Makefile:38]: Have you tried to use our preferred jpeg implementation? You just need to add# build with the distribution preferred libjpeg implementation
CFLAGS   += $(JPEG_CPPFLAGS) $(JPEG_CFLAGS)
CXXFLAGS += $(JPEG_CPPFLAGS) $(JPEG_CXXFLAGS)
LDFLAGS  += $(JPEG_LDFLAGS)
CONFIGURE_ENV += CPPFLAGS="$(JPEG_CPPFLAGS)"

here.

**Comment by fritzkink** [comment]
Ok, I removed the 64bit setting and also changed the jpeg libs to the preferred ones (thanks for the hint). 
With respect to the tests - those need extra packages like netpbm:
'configure: error: "Cannot make tests without pbmtog3 installed (does your system require a netpbm-progs package?)"
make: *** [/usr/share/src/oi-userland/make-rules/configure.mk:166: /usr/src/oi-userland/components/multimedia/spandsp/build/amd64/.configured] Error 1
'
which I didn't want to create. Is it ok?

**Pull request merged**